[2024-04-17T12:00:16.789+0000] {processor.py:154} INFO - Started process (PID=4635) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:00:16.801+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:00:16.805+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:00:16.805+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:00:17.010+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:00:16.958+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:00:17.028+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:00:17.158+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.405 seconds
[2024-04-17T12:00:48.243+0000] {processor.py:154} INFO - Started process (PID=4707) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:00:48.246+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:00:48.255+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:00:48.254+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:00:48.302+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:00:48.295+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:00:48.306+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:00:48.347+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.112 seconds
[2024-04-17T12:01:18.694+0000] {processor.py:154} INFO - Started process (PID=4770) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:01:18.696+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:01:18.698+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:01:18.697+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:01:18.712+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:01:18.710+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:01:18.713+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:01:18.750+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.063 seconds
[2024-04-17T12:01:49.039+0000] {processor.py:154} INFO - Started process (PID=4843) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:01:49.043+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:01:49.044+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:01:49.044+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:01:49.061+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:01:49.057+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:01:49.063+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:01:49.110+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.079 seconds
[2024-04-17T12:02:19.704+0000] {processor.py:154} INFO - Started process (PID=4912) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:02:19.706+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:02:19.709+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:02:19.708+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:02:19.723+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:02:19.721+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:02:19.725+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:02:19.767+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.071 seconds
[2024-04-17T12:02:50.399+0000] {processor.py:154} INFO - Started process (PID=4988) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:02:50.403+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:02:50.405+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:02:50.405+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:02:50.419+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:02:50.417+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:02:50.421+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:02:50.458+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.066 seconds
[2024-04-17T12:03:21.188+0000] {processor.py:154} INFO - Started process (PID=5057) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:03:21.191+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:03:21.194+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:03:21.194+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:03:21.211+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:03:21.209+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:03:21.213+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:03:21.249+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.070 seconds
[2024-04-17T12:03:51.568+0000] {processor.py:154} INFO - Started process (PID=5135) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:03:51.571+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:03:51.610+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:03:51.574+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:03:51.681+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:03:51.674+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:03:51.684+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:03:51.758+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.195 seconds
[2024-04-17T12:04:21.900+0000] {processor.py:154} INFO - Started process (PID=5204) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:04:21.903+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:04:21.905+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:04:21.904+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:04:21.922+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:04:21.920+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:04:21.923+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:04:21.961+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.069 seconds
[2024-04-17T12:04:52.078+0000] {processor.py:154} INFO - Started process (PID=5282) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:04:52.123+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:04:52.125+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:04:52.125+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:04:52.149+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:04:52.146+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:04:52.151+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:04:52.201+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.133 seconds
[2024-04-17T12:05:22.290+0000] {processor.py:154} INFO - Started process (PID=5350) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:05:22.295+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:05:22.299+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:05:22.298+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:05:22.339+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:05:22.334+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:05:22.342+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:05:22.396+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.117 seconds
[2024-04-17T12:05:52.554+0000] {processor.py:154} INFO - Started process (PID=5429) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:05:52.556+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:05:52.558+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:05:52.557+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:05:52.575+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:05:52.572+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:05:52.576+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:05:52.611+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.063 seconds
[2024-04-17T12:06:23.405+0000] {processor.py:154} INFO - Started process (PID=5499) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:06:23.407+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:06:23.410+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:06:23.409+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:06:23.468+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:06:23.460+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:06:23.470+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:06:23.545+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.151 seconds
[2024-04-17T12:06:54.173+0000] {processor.py:154} INFO - Started process (PID=5576) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:06:54.176+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:06:54.178+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:06:54.177+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:06:54.196+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:06:54.193+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:06:54.197+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:06:54.239+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.075 seconds
[2024-04-17T12:07:24.553+0000] {processor.py:154} INFO - Started process (PID=5643) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:07:24.556+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:07:24.558+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:07:24.558+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:07:24.578+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:07:24.575+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:07:24.580+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:07:24.624+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.079 seconds
[2024-04-17T12:07:55.622+0000] {processor.py:154} INFO - Started process (PID=5719) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:07:55.625+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:07:55.627+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:07:55.627+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:07:55.653+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:07:55.650+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:07:55.655+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:07:55.727+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.116 seconds
[2024-04-17T12:08:26.115+0000] {processor.py:154} INFO - Started process (PID=5788) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:08:26.120+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:08:26.123+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:08:26.123+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:08:26.148+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:08:26.145+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:08:26.151+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:08:26.195+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.096 seconds
[2024-04-17T12:08:56.674+0000] {processor.py:154} INFO - Started process (PID=5856) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:08:56.677+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:08:56.679+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:08:56.678+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:08:56.698+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:08:56.695+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:08:56.700+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:08:56.742+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.078 seconds
[2024-04-17T12:09:26.860+0000] {processor.py:154} INFO - Started process (PID=5935) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:09:26.863+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:09:26.865+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:09:26.865+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:09:26.882+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:09:26.880+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:09:26.884+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:09:26.923+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.073 seconds
[2024-04-17T12:09:57.255+0000] {processor.py:154} INFO - Started process (PID=6003) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:09:57.259+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:09:57.261+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:09:57.261+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:09:57.279+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:09:57.276+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:09:57.281+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:09:57.339+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.091 seconds
[2024-04-17T12:10:27.535+0000] {processor.py:154} INFO - Started process (PID=6077) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:10:27.538+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:10:27.541+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:10:27.540+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:10:27.563+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:10:27.560+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:10:27.565+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:10:27.626+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.103 seconds
[2024-04-17T12:10:58.377+0000] {processor.py:154} INFO - Started process (PID=6146) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:10:58.380+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:10:58.382+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:10:58.382+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:10:58.401+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:10:58.398+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:10:58.404+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:10:58.441+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.071 seconds
[2024-04-17T12:11:28.804+0000] {processor.py:154} INFO - Started process (PID=6223) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:11:28.806+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:11:28.810+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:11:28.810+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:11:28.832+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:11:28.829+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:11:28.834+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:11:28.884+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.090 seconds
[2024-04-17T12:11:59.402+0000] {processor.py:154} INFO - Started process (PID=6292) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:11:59.404+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:11:59.406+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:11:59.406+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:11:59.425+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:11:59.422+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:11:59.427+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:11:59.468+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.078 seconds
[2024-04-17T12:12:30.076+0000] {processor.py:154} INFO - Started process (PID=6371) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:12:30.078+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:12:30.080+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:12:30.080+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:12:30.099+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:12:30.096+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:12:30.101+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:12:30.145+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.076 seconds
[2024-04-17T12:13:00.792+0000] {processor.py:154} INFO - Started process (PID=6440) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:13:00.800+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:13:00.809+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:13:00.809+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:13:00.833+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:13:00.829+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:13:00.835+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:13:00.881+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.114 seconds
[2024-04-17T12:13:30.980+0000] {processor.py:154} INFO - Started process (PID=6520) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:13:30.983+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:13:30.984+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:13:30.984+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:13:31.004+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:13:31.002+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:13:31.006+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:13:31.055+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.084 seconds
[2024-04-17T12:14:01.411+0000] {processor.py:154} INFO - Started process (PID=6584) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:14:01.421+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:14:01.425+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:14:01.425+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:14:01.466+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:14:01.458+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:14:01.473+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:14:01.537+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.141 seconds
[2024-04-17T12:14:31.842+0000] {processor.py:154} INFO - Started process (PID=6662) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:14:31.844+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:14:31.847+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:14:31.846+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:14:31.867+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:14:31.864+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:14:31.870+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:14:31.913+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.080 seconds
[2024-04-17T12:15:02.051+0000] {processor.py:154} INFO - Started process (PID=6729) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:15:02.056+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:15:02.059+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:15:02.059+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:15:02.086+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:15:02.082+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:15:02.087+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:15:02.139+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.099 seconds
[2024-04-17T12:15:32.269+0000] {processor.py:154} INFO - Started process (PID=6797) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:15:32.272+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:15:32.274+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:15:32.274+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:15:32.295+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:15:32.290+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:15:32.297+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:15:32.342+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.089 seconds
[2024-04-17T12:16:03.029+0000] {processor.py:154} INFO - Started process (PID=6875) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:16:03.034+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:16:03.042+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:16:03.042+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:16:03.088+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:16:03.080+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:16:03.091+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:16:03.180+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.160 seconds
[2024-04-17T12:16:33.380+0000] {processor.py:154} INFO - Started process (PID=6944) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:16:33.383+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:16:33.386+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:16:33.385+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:16:33.403+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:16:33.399+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:16:33.404+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:16:33.447+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.074 seconds
[2024-04-17T12:17:03.615+0000] {processor.py:154} INFO - Started process (PID=7023) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:17:03.625+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:17:03.628+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:17:03.628+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:17:03.664+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:17:03.659+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:17:03.669+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:17:03.738+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.138 seconds
[2024-04-17T12:17:34.363+0000] {processor.py:154} INFO - Started process (PID=7092) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:17:34.366+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:17:34.371+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:17:34.370+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:17:34.395+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:17:34.391+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:17:34.396+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:17:34.444+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.092 seconds
[2024-04-17T12:18:05.008+0000] {processor.py:154} INFO - Started process (PID=7170) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:18:05.027+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:18:05.030+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:18:05.030+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:18:05.074+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:18:05.067+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:18:05.079+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:18:05.134+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.154 seconds
[2024-04-17T12:18:35.441+0000] {processor.py:154} INFO - Started process (PID=7237) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:18:35.443+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:18:35.446+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:18:35.445+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:18:35.544+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:18:35.539+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:18:35.547+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:18:35.610+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.180 seconds
[2024-04-17T12:19:05.819+0000] {processor.py:154} INFO - Started process (PID=7306) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:19:05.823+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:19:05.827+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:19:05.827+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:19:05.851+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:19:05.847+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:19:05.853+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:19:05.918+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.111 seconds
[2024-04-17T12:19:36.742+0000] {processor.py:154} INFO - Started process (PID=7383) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:19:36.749+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:19:36.752+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:19:36.752+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:19:36.774+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:19:36.771+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:19:36.776+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:19:36.829+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.100 seconds
[2024-04-17T12:20:07.173+0000] {processor.py:154} INFO - Started process (PID=7452) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:20:07.176+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:20:07.179+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:20:07.178+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:20:07.200+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:20:07.196+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:20:07.202+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:20:07.259+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.097 seconds
[2024-04-17T12:20:37.984+0000] {processor.py:154} INFO - Started process (PID=7522) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:20:37.987+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:20:37.990+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:20:37.989+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:20:38.035+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:20:38.028+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:20:38.038+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:20:38.116+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.144 seconds
[2024-04-17T12:21:08.270+0000] {processor.py:154} INFO - Started process (PID=7599) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:21:08.273+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:21:08.279+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:21:08.277+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:21:08.304+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:21:08.301+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:21:08.306+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:21:08.372+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.114 seconds
[2024-04-17T12:21:38.503+0000] {processor.py:154} INFO - Started process (PID=7664) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:21:38.505+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:21:38.509+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:21:38.509+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:21:38.531+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:21:38.526+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:21:38.533+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:21:38.589+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.098 seconds
[2024-04-17T12:22:09.407+0000] {processor.py:154} INFO - Started process (PID=7732) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:22:09.412+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:22:09.415+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:22:09.414+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:22:09.463+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:22:09.458+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:22:09.465+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:22:09.543+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.150 seconds
[2024-04-17T12:22:40.263+0000] {processor.py:154} INFO - Started process (PID=7807) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:22:40.266+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:22:40.270+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:22:40.269+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:22:40.296+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:22:40.292+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:22:40.297+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:22:40.352+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.103 seconds
[2024-04-17T12:23:10.552+0000] {processor.py:154} INFO - Started process (PID=7877) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:23:10.564+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:23:10.567+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:23:10.566+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:23:10.599+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:23:10.593+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:23:10.636+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:23:10.752+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.229 seconds
[2024-04-17T12:23:41.288+0000] {processor.py:154} INFO - Started process (PID=7946) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:23:41.291+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:23:41.293+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:23:41.293+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:23:41.316+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:23:41.313+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:23:41.318+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:23:41.360+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.078 seconds
[2024-04-17T12:24:11.768+0000] {processor.py:154} INFO - Started process (PID=8015) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:24:11.771+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:24:11.775+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:24:11.775+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:24:11.820+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:24:11.815+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:24:11.822+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:24:11.869+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.114 seconds
[2024-04-17T12:24:42.187+0000] {processor.py:154} INFO - Started process (PID=8086) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:24:42.190+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:24:42.191+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:24:42.191+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:24:42.213+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:24:42.209+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:24:42.218+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:24:42.261+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.086 seconds
[2024-04-17T12:25:12.700+0000] {processor.py:154} INFO - Started process (PID=8154) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:25:12.704+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:25:12.709+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:25:12.709+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:25:12.735+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:25:12.731+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:25:12.737+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:25:12.791+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.103 seconds
[2024-04-17T12:25:44.586+0000] {processor.py:154} INFO - Started process (PID=8231) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:25:44.592+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:25:44.598+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:25:44.597+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:25:44.661+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:25:44.648+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:25:44.668+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:25:44.768+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.200 seconds
[2024-04-17T12:26:15.653+0000] {processor.py:154} INFO - Started process (PID=8302) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:26:15.657+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:26:15.666+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:26:15.662+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:26:15.740+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:26:15.736+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:26:15.742+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:26:15.989+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.377 seconds
[2024-04-17T12:26:46.288+0000] {processor.py:154} INFO - Started process (PID=8368) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:26:46.294+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:26:46.299+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:26:46.298+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:26:46.343+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:26:46.331+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:26:46.346+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:26:46.439+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.165 seconds
[2024-04-17T12:27:16.853+0000] {processor.py:154} INFO - Started process (PID=8435) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:27:16.874+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:27:16.878+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:27:16.878+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:27:16.945+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:27:16.931+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:27:16.948+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:27:17.063+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.223 seconds
[2024-04-17T12:27:47.311+0000] {processor.py:154} INFO - Started process (PID=8503) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:27:47.314+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:27:47.319+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:27:47.317+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:27:47.350+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:27:47.345+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:27:47.353+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:27:47.428+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.130 seconds
[2024-04-17T12:28:18.136+0000] {processor.py:154} INFO - Started process (PID=8569) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:28:18.139+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:28:18.144+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:28:18.143+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:28:18.170+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:28:18.168+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:28:18.172+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:28:18.227+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.103 seconds
[2024-04-17T12:28:48.578+0000] {processor.py:154} INFO - Started process (PID=8637) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:28:48.581+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:28:48.585+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:28:48.584+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:28:48.609+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:28:48.606+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:28:48.611+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:28:48.675+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.105 seconds
[2024-04-17T12:29:19.108+0000] {processor.py:154} INFO - Started process (PID=8714) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:29:19.111+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:29:19.115+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:29:19.114+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:29:19.146+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:29:19.142+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:29:19.148+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:29:19.212+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.122 seconds
[2024-04-17T12:29:49.783+0000] {processor.py:154} INFO - Started process (PID=8785) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:29:49.786+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:29:49.790+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:29:49.788+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:29:49.820+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:29:49.816+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:29:49.822+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:29:49.883+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.109 seconds
[2024-04-17T12:30:19.973+0000] {processor.py:154} INFO - Started process (PID=8854) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:30:19.979+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:30:19.982+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:30:19.982+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:30:20.012+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:30:20.006+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:30:20.015+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:30:20.083+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.128 seconds
[2024-04-17T12:30:50.565+0000] {processor.py:154} INFO - Started process (PID=8924) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:30:50.570+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:30:50.575+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:30:50.575+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:30:50.624+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:30:50.620+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:30:50.626+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:30:50.705+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.151 seconds
[2024-04-17T12:31:21.114+0000] {processor.py:154} INFO - Started process (PID=9004) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:31:21.118+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:31:21.121+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:31:21.121+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:31:21.158+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:31:21.154+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:31:21.161+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:31:21.228+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.130 seconds
[2024-04-17T12:31:51.472+0000] {processor.py:154} INFO - Started process (PID=9071) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:31:51.475+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:31:51.477+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:31:51.477+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:31:51.501+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:31:51.498+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:31:51.504+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:31:51.554+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.091 seconds
[2024-04-17T12:32:21.718+0000] {processor.py:154} INFO - Started process (PID=9140) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:32:21.721+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:32:21.723+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:32:21.722+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:32:21.741+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:32:21.737+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:32:21.742+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:32:21.779+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.067 seconds
[2024-04-17T12:32:51.963+0000] {processor.py:154} INFO - Started process (PID=9218) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:32:51.976+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:32:51.982+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:32:51.981+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:32:52.006+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:32:52.002+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:32:52.010+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:32:52.108+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.159 seconds
[2024-04-17T12:33:22.571+0000] {processor.py:154} INFO - Started process (PID=9285) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:33:22.574+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:33:22.576+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:33:22.576+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:33:22.601+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:33:22.594+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:33:22.603+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:33:22.653+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.102 seconds
[2024-04-17T12:33:53.140+0000] {processor.py:154} INFO - Started process (PID=9364) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:33:53.158+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:33:53.161+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:33:53.161+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:33:53.208+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:33:53.200+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:33:53.210+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:33:53.285+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.170 seconds
[2024-04-17T12:34:23.895+0000] {processor.py:154} INFO - Started process (PID=9432) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:34:23.900+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:34:23.911+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:34:23.910+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:34:23.967+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:34:23.953+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:34:23.976+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:34:24.051+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.172 seconds
[2024-04-17T12:34:55.091+0000] {processor.py:154} INFO - Started process (PID=9502) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:34:55.098+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:34:55.100+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:34:55.100+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:34:55.122+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:34:55.119+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:34:55.126+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:34:55.185+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.102 seconds
[2024-04-17T12:35:25.952+0000] {processor.py:154} INFO - Started process (PID=9579) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:35:25.956+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:35:25.966+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:35:25.966+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:35:26.001+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:35:25.998+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:35:26.004+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:35:26.070+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.153 seconds
[2024-04-17T12:35:56.664+0000] {processor.py:154} INFO - Started process (PID=9650) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:35:56.694+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:35:56.698+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:35:56.697+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:35:56.770+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:35:56.757+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:35:56.774+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:35:56.886+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.245 seconds
[2024-04-17T12:36:27.096+0000] {processor.py:154} INFO - Started process (PID=9716) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:36:27.099+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:36:27.101+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:36:27.100+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:36:27.119+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:36:27.116+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:36:27.121+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:36:27.159+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.069 seconds
[2024-04-17T12:36:57.429+0000] {processor.py:154} INFO - Started process (PID=9793) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:36:57.431+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:36:57.434+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:36:57.434+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:36:57.462+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:36:57.458+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:36:57.464+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:36:57.511+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.095 seconds
[2024-04-17T12:37:28.151+0000] {processor.py:154} INFO - Started process (PID=9863) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:37:28.155+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:37:28.158+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:37:28.158+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:37:28.219+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:37:28.216+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:37:28.223+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:37:28.326+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.198 seconds
[2024-04-17T12:37:58.663+0000] {processor.py:154} INFO - Started process (PID=9941) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:37:58.668+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:37:58.670+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:37:58.669+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:37:58.703+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:37:58.695+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:37:58.704+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:37:58.754+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.100 seconds
[2024-04-17T12:38:29.041+0000] {processor.py:154} INFO - Started process (PID=10008) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:38:29.043+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:38:29.045+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:38:29.045+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:38:29.069+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:38:29.066+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:38:29.071+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:38:29.115+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.082 seconds
[2024-04-17T12:38:59.525+0000] {processor.py:154} INFO - Started process (PID=10075) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:38:59.527+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:38:59.530+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:38:59.529+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:38:59.549+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:38:59.546+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:38:59.551+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:38:59.590+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.071 seconds
[2024-04-17T12:39:30.034+0000] {processor.py:154} INFO - Started process (PID=10153) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:39:30.038+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:39:30.040+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:39:30.040+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:39:30.072+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:39:30.065+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:39:30.081+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:39:30.190+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.169 seconds
[2024-04-17T12:40:00.486+0000] {processor.py:154} INFO - Started process (PID=10222) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:40:00.490+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:40:00.492+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:40:00.492+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:40:00.521+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:40:00.517+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:40:00.523+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:40:00.570+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.145 seconds
[2024-04-17T12:40:30.789+0000] {processor.py:154} INFO - Started process (PID=10290) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:40:30.791+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:40:30.793+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:40:30.793+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:40:30.813+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:40:30.809+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:40:30.816+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:40:30.857+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.076 seconds
[2024-04-17T12:41:01.344+0000] {processor.py:154} INFO - Started process (PID=10359) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:41:01.346+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:41:01.349+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:41:01.348+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:41:01.370+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:41:01.366+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:41:01.371+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:41:01.411+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.073 seconds
[2024-04-17T12:41:31.561+0000] {processor.py:154} INFO - Started process (PID=10436) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:41:31.564+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:41:31.568+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:41:31.567+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:41:31.605+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:41:31.597+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:41:31.612+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:41:31.674+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.122 seconds
[2024-04-17T12:42:02.137+0000] {processor.py:154} INFO - Started process (PID=10505) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:42:02.140+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:42:02.146+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:42:02.145+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:42:02.173+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:42:02.169+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:42:02.182+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:42:02.231+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.107 seconds
[2024-04-17T12:42:32.302+0000] {processor.py:154} INFO - Started process (PID=10574) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:42:32.305+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:42:32.307+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:42:32.307+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:42:32.330+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:42:32.326+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:42:32.332+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:42:32.377+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.088 seconds
[2024-04-17T12:43:02.458+0000] {processor.py:154} INFO - Started process (PID=10650) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:43:02.460+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:43:02.462+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:43:02.462+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:43:02.479+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:43:02.477+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:43:02.481+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:43:02.522+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.073 seconds
[2024-04-17T12:43:32.968+0000] {processor.py:154} INFO - Started process (PID=10719) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:43:32.970+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:43:32.971+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:43:32.971+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:43:32.986+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:43:32.983+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:43:32.988+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:43:33.018+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.057 seconds
[2024-04-17T12:44:03.591+0000] {processor.py:154} INFO - Started process (PID=10795) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:44:03.593+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:44:03.596+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:44:03.595+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:44:03.615+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:44:03.611+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:44:03.616+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:44:03.659+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.076 seconds
[2024-04-17T12:44:33.750+0000] {processor.py:154} INFO - Started process (PID=10864) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:44:33.753+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:44:33.759+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:44:33.758+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:44:33.782+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:44:33.778+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:44:33.783+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:44:33.831+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.087 seconds
[2024-04-17T12:45:04.470+0000] {processor.py:154} INFO - Started process (PID=10933) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:45:04.473+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:45:04.480+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:45:04.480+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:45:04.502+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:45:04.498+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:45:04.503+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:45:04.554+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.090 seconds
[2024-04-17T12:45:34.936+0000] {processor.py:154} INFO - Started process (PID=11011) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:45:34.941+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:45:34.944+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:45:34.943+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:45:34.976+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:45:34.972+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:45:34.980+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:45:35.036+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.113 seconds
[2024-04-17T12:46:05.376+0000] {processor.py:154} INFO - Started process (PID=11081) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:46:05.378+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:46:05.380+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:46:05.380+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:46:05.399+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:46:05.396+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:46:05.401+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:46:05.435+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.065 seconds
[2024-04-17T12:46:35.503+0000] {processor.py:154} INFO - Started process (PID=11159) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:46:35.506+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:46:35.509+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:46:35.508+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:46:35.527+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:46:35.523+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:46:35.528+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:46:35.566+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.071 seconds
[2024-04-17T12:47:06.226+0000] {processor.py:154} INFO - Started process (PID=11229) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:47:06.229+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:47:06.232+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:47:06.232+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:47:06.266+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:47:06.262+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:47:06.268+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:47:06.322+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.111 seconds
[2024-04-17T12:47:36.563+0000] {processor.py:154} INFO - Started process (PID=11307) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:47:36.565+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:47:36.567+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:47:36.567+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:47:36.589+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:47:36.585+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:47:36.590+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:47:36.633+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.078 seconds
[2024-04-17T12:48:07.221+0000] {processor.py:154} INFO - Started process (PID=11373) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:48:07.223+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:48:07.226+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:48:07.226+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:48:07.259+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:48:07.256+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:48:07.261+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:48:07.321+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.111 seconds
[2024-04-17T12:48:37.537+0000] {processor.py:154} INFO - Started process (PID=11443) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:48:37.539+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:48:37.541+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:48:37.541+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:48:37.560+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:48:37.557+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:48:37.562+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:48:37.602+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.072 seconds
[2024-04-17T12:49:07.920+0000] {processor.py:154} INFO - Started process (PID=11521) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:49:07.926+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:49:07.929+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:49:07.928+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:49:07.965+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:49:07.961+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:49:07.967+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:49:08.042+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.142 seconds
[2024-04-17T12:49:38.580+0000] {processor.py:154} INFO - Started process (PID=11590) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:49:38.583+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:49:38.585+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:49:38.584+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:49:38.604+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:49:38.601+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:49:38.605+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:49:38.656+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.086 seconds
[2024-04-17T12:50:09.290+0000] {processor.py:154} INFO - Started process (PID=11659) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:50:09.299+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:50:09.301+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:50:09.301+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:50:09.335+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:50:09.331+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:50:09.337+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:50:09.408+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.127 seconds
[2024-04-17T12:50:40.034+0000] {processor.py:154} INFO - Started process (PID=11737) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:50:40.039+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:50:40.046+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:50:40.045+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:50:40.077+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:50:40.072+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:50:40.084+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:50:40.169+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.147 seconds
[2024-04-17T12:51:12.236+0000] {processor.py:154} INFO - Started process (PID=11806) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:51:12.256+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:51:12.260+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:51:12.260+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:51:12.328+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:51:12.313+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:51:12.330+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:51:12.441+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.215 seconds
[2024-04-17T12:51:42.652+0000] {processor.py:154} INFO - Started process (PID=11873) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:51:42.657+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:51:42.660+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:51:42.660+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:51:42.686+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:51:42.682+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:51:42.688+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:51:42.762+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.125 seconds
[2024-04-17T12:52:13.418+0000] {processor.py:154} INFO - Started process (PID=11939) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:52:13.421+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:52:13.425+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:52:13.424+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:52:13.462+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:52:13.456+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:52:13.465+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:52:13.529+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.121 seconds
[2024-04-17T12:52:44.451+0000] {processor.py:154} INFO - Started process (PID=12017) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:52:44.455+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:52:44.460+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:52:44.459+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:52:44.494+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:52:44.489+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:52:44.497+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:52:44.556+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.118 seconds
[2024-04-17T12:53:14.967+0000] {processor.py:154} INFO - Started process (PID=12084) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:53:14.977+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:53:14.981+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:53:14.981+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:53:15.009+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:53:15.005+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:53:15.011+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:53:15.082+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.123 seconds
[2024-04-17T12:53:45.170+0000] {processor.py:154} INFO - Started process (PID=12151) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:53:45.180+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:53:45.186+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:53:45.186+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:53:45.237+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:53:45.230+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:53:45.242+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:53:45.304+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.151 seconds
[2024-04-17T12:54:15.497+0000] {processor.py:154} INFO - Started process (PID=12217) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:54:15.501+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:54:15.504+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:54:15.503+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:54:15.537+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:54:15.533+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:54:15.539+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:54:15.610+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.127 seconds
[2024-04-17T12:54:46.342+0000] {processor.py:154} INFO - Started process (PID=12283) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:54:46.346+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:54:46.351+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:54:46.350+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:54:46.392+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:54:46.389+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:54:46.394+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:54:46.462+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.137 seconds
[2024-04-17T12:55:16.826+0000] {processor.py:154} INFO - Started process (PID=12349) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:55:16.833+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:55:16.847+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:55:16.841+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:55:16.907+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:55:16.902+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:55:16.926+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:55:17.038+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.228 seconds
[2024-04-17T12:55:47.751+0000] {processor.py:154} INFO - Started process (PID=12425) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:55:47.761+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:55:47.764+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:55:47.763+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:55:47.819+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:55:47.799+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:55:47.823+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:55:47.888+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.155 seconds
[2024-04-17T12:56:18.659+0000] {processor.py:154} INFO - Started process (PID=12495) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:56:18.664+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:56:18.668+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:56:18.668+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:56:18.732+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:56:18.721+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:56:18.740+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:56:18.830+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.194 seconds
[2024-04-17T12:56:49.279+0000] {processor.py:154} INFO - Started process (PID=12565) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:56:49.283+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:56:49.286+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:56:49.286+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:56:49.312+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:56:49.307+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:56:49.315+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:56:49.358+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.090 seconds
[2024-04-17T12:57:19.718+0000] {processor.py:154} INFO - Started process (PID=12633) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:57:19.721+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:57:19.723+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:57:19.723+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:57:19.751+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:57:19.744+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:57:19.753+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:57:19.807+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.103 seconds
[2024-04-17T12:57:50.091+0000] {processor.py:154} INFO - Started process (PID=12702) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:57:50.093+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:57:50.096+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:57:50.095+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:57:50.114+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:57:50.111+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:57:50.117+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:57:50.159+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.078 seconds
[2024-04-17T12:58:20.529+0000] {processor.py:154} INFO - Started process (PID=12780) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:58:20.532+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:58:20.535+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:58:20.534+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:58:20.556+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:58:20.552+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:58:20.557+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:58:20.594+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.072 seconds
[2024-04-17T12:58:51.369+0000] {processor.py:154} INFO - Started process (PID=12849) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:58:51.372+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:58:51.375+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:58:51.374+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:58:51.399+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:58:51.396+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:58:51.401+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:58:51.447+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.084 seconds
[2024-04-17T12:59:21.592+0000] {processor.py:154} INFO - Started process (PID=12918) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:59:21.596+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:59:21.598+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:59:21.598+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:59:21.629+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:59:21.624+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:59:21.633+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:59:21.856+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.275 seconds
[2024-04-17T12:59:52.154+0000] {processor.py:154} INFO - Started process (PID=12986) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:59:52.157+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T12:59:52.162+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:59:52.162+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:59:52.195+0000] {logging_mixin.py:120} INFO - [2024-04-17T12:59:52.191+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T12:59:52.197+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T12:59:52.274+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.130 seconds
[2024-04-17T13:00:22.713+0000] {processor.py:154} INFO - Started process (PID=13054) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:00:22.718+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:00:22.723+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:00:22.722+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:00:22.775+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:00:22.766+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:00:22.778+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:00:22.849+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.153 seconds
[2024-04-17T13:00:53.373+0000] {processor.py:154} INFO - Started process (PID=13132) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:00:53.382+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:00:53.385+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:00:53.385+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:00:53.426+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:00:53.418+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:00:53.430+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:00:53.518+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.158 seconds
[2024-04-17T13:01:23.715+0000] {processor.py:154} INFO - Started process (PID=13199) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:01:23.721+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:01:23.748+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:01:23.747+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:01:23.818+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:01:23.812+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:01:23.823+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:01:23.889+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.189 seconds
[2024-04-17T13:01:54.024+0000] {processor.py:154} INFO - Started process (PID=13266) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:01:54.030+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:01:54.034+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:01:54.034+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:01:54.081+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:01:54.071+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:01:54.092+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:01:54.236+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.225 seconds
[2024-04-17T13:02:24.975+0000] {processor.py:154} INFO - Started process (PID=13335) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:02:24.979+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:02:24.982+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:02:24.982+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:02:25.027+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:02:25.023+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:02:25.030+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:02:25.108+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.146 seconds
[2024-04-17T13:02:55.509+0000] {processor.py:154} INFO - Started process (PID=13404) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:02:55.511+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:02:55.514+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:02:55.514+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:02:55.551+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:02:55.546+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:02:55.557+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:02:55.617+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.123 seconds
[2024-04-17T13:03:25.738+0000] {processor.py:154} INFO - Started process (PID=13483) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:03:25.746+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:03:25.757+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:03:25.756+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:03:25.814+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:03:25.801+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:03:25.818+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:03:25.901+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.175 seconds
[2024-04-17T13:03:56.500+0000] {processor.py:154} INFO - Started process (PID=13550) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:03:56.504+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:03:56.509+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:03:56.509+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:03:56.553+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:03:56.549+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:03:56.556+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:03:56.633+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.146 seconds
[2024-04-17T13:04:26.855+0000] {processor.py:154} INFO - Started process (PID=13617) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:04:26.857+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:04:26.860+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:04:26.860+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:04:26.886+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:04:26.883+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:04:26.887+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:04:26.979+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.134 seconds
[2024-04-17T13:04:57.274+0000] {processor.py:154} INFO - Started process (PID=13686) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:04:57.278+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-17T13:04:57.280+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:04:57.280+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:04:57.302+0000] {logging_mixin.py:120} INFO - [2024-04-17T13:04:57.298+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-17T13:04:57.303+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-17T13:04:57.347+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.082 seconds
